Tests/LibPDF/TestPDF.cpp

```diff
@@ -29,3 +31,17 @@ TEST_CASE(complex_pdf)
     auto document = PDF::Document::create(file->bytes());
     EXPECT_EQ(document->get_page_count(), 3U);
 }
+
+TEST_CASE(empty_file_issue_10702)
+{
+    AK::ReadonlyBytes empty;
+    auto document = PDF::Document::create(empty);
+    EXPECT(document.is_null());
+}
+
+TEST_CASE(truncated_pdf_header_issue_10717)
+{
+    AK::String string { "%PDF-2.11%" };
+    auto document = PDF::Document::create(string.bytes());
```

Comment:
I didn't realize that the "test cases" are so short. Nevermind then! Putting them in-line is a good idea, please forget what I said initially :D